starburst-presto/presto.sh

```diff
@@ -229,6 +249,9 @@ EOF
 function configure_and_start_presto() {
   mkdir -p /opt/presto-server/etc/catalog
 
+  update_apt_get
+  install_apt_get openjdk-11-jre
```

Comment:
May you instead set property to preserve compatibility w/ Java 8? Generally, on Dataproc we try to use the same JVM for all components to simplify deployment and maintenance.

Reply:
> property to preserve compatibility w/ Java 8?

Okay!.

Reply:
@ChethanUK please add `-Dpresto-temporarily-allow-java8=true` to `jvm.config`

Reply:
Yes will give the option to choose Java8 as well.

Reply:
JDK 11 removed